src/themes/light.scss

```diff
@@ -183,4 +183,6 @@
   --wxc-tooltip-background: #{$wxc-white-alpha-100};
   --wxc-tooltip-border: 1px solid #{$wxc-black-alpha-20};
   --wxc-tooltip-border-color: #{$wxc-black-alpha-20};
+
+  --wxc--focus--box-shadow: 0 0 0 0.25rem #{$wxc-blue-60-alpha-30}, 0 0 0 0.125rem #{$wxc-blue-60};
```

Comment:
put this at the top with the other generic variables, eg after `--wxc-box-shadow` on line 16